app/models.py

```diff
@@ -0,0 +1,133 @@
+# This is where you define the models of your application.
+# This may be split into several modules in the same way as views.py.
+
+from datetime import datetime
+from werkzeug.security import generate_password_hash, \
+     check_password_hash
+from flask_login import UserMixin
+from itsdangerous import (TimedJSONWebSignatureSerializer
+                          as Serializer, BadSignature, SignatureExpired)
+
+from app import app, create_app
+
+db = create_app()
```

Comment:
I'm not sure you want to do this here. You have an instance here which you could easily have imported from your `__init__.py` file